treenav/tests/test_templatetags.py

```diff
@@ -0,0 +1,304 @@
+from unittest.mock import ANY, patch
```

Comment:
Yes, this would fail. I was more thinking about creating a package update instead of merging this to the stable version. What I mean is that I was more thinking about dropping support for py2.7 since it is out-of-date since 01-01-2020. But if you think we should still support python < 3.3 then I can suggest installing the 3rd party [mock](https://pypi.org/project/mock/) package. 

Btw. I think we should talk about the DjTreenav update a little more in detail because there are some things that require agreements. I don't want to start something without discussing it with you guys :smiley: 

We could then create some points to prepare the path for the next major (I would call this major because since this package wasn't updated for 3years it needs to get ready for the existing Django versions any new python version (3.7 - 3.8) update. Thanks to this the upgrade path would be more clarified.

Reply:
Sounds good to me.